tests/test_processors.py

```diff
@@ -456,6 +456,17 @@ def test_obtains_exc_info_on_True(self):
         else:
             pytest.fail("Exception not raised.")
 
+    def test_obtains_exc_info_on_number(self):
```

Comment:
No problem, I didn't know about that extra on pytest, maybe I should make myself more familiar with it. Thanks for the hint )))